mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FaqScreen.kt

```diff
@@ -0,0 +1,49 @@
+package org.mifos.mobilewallet.mifospay.faq.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.designsystem.component.FaqItemScreen
+import org.mifos.mobilewallet.mifospay.designsystem.component.MifosTopBar
+import org.mifos.mobilewallet.mifospay.faq.presenter.FAQViewModel
+
+@Composable
+fun FaqScreen(
+    navigateBack: () -> Unit,
+    faqViewModel: FAQViewModel = hiltViewModel(),
+    updateFaqPosition: (Int) -> Unit,
+) {
+    Column(modifier = Modifier.fillMaxSize()) {
+        MifosTopBar(
+            topBarTitle = R.string.frequently_asked_questions,
+            backPress = { navigateBack.invoke() })
+        LazyColumn(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxWidth()
+        ) {
+            itemsIndexed(items = faqViewModel.getFAQ()) { index, faqItem ->
+                FaqItemScreen(
+                    question = stringResource(id = faqItem.question),
+                    answer = faqItem.answer?.let { stringResource(id = it) },
+                    onItemSelected = { position -> updateFaqPosition(position) },
+                    index = index
+                )
+            }
+        }
+    }
+}
+
```

Comment:
@PratyushSingh07 Add one more function with the same name 
```
Composable
fun FaqScreen(
  navigateBack: () -> Unit,
  faqList: List<FAQ>
) {
Column(modifier = Modifier.fillMaxSize()) {
        MifosTopBar(
            topBarTitle = R.string.frequently_asked_questions,
            backPress = { navigateBack.invoke() })
        LazyColumn(
            modifier = Modifier
                .weight(1f)
                .fillMaxWidth()
        ) {
            itemsIndexed(items = faqList) { index, faqItem ->
                FaqItemScreen(
                    question = stringResource(id = faqItem.question),
                    answer = faqItem.answer?.let { stringResource(id = it) },
                    onItemSelected = { position -> updateFaqPosition(position) },
                    index = index
                )
            }
        }
    }
}
```

And call it in above function. So now you can generate preview of function with faqList passing

Reply:
Hey @therajanmaurya , I have done rest of the changes except this one. My doubt was that are we overloading this function only for the purpose of preview ? If yes, then I dont think we need to do that because the preview is being generated in the current scenario. 
![image](https://github.com/openMF/mobile-wallet/assets/90026952/70e66116-4d49-4404-bea7-961bad314fb6)

Reply:
It is also make UI test possible that's why I suggested the change.